client/tests/webdriver/customFieldsSpecs/showTask.spec.js

```diff
@@ -30,4 +30,44 @@ describe("Show task page", () => {
       expect(await question1AssessmentWeekly.isExisting()).to.be.true
     })
   })
+
+  describe("When on the show page of a task", () => {
+    it("We should see a parent task if there is any related to the current task", async() => {
+      const parentTask = await (await ShowTask.getParentTask())
+      const parentTaskField = await (await ShowTask.getParentTaskField())
+      if (await parentTask.isExisting()) {
+        await (await ShowTask.getParentTaskField()).waitForDisplayed()
+        // eslint-disable-next-line no-unused-expressions
+        expect(await parentTaskField.isExisting()).to.be.true
+      }
+    })
+
+    it("We should not see a parent task if there is not any related to the current task", async() => {
+      const parentTask = await (await ShowTask.getParentTask())
+      const parentTaskField = await (await ShowTask.getParentTaskField())
+      if (await parentTask.isExisting() === false) {
+        // eslint-disable-next-line no-unused-expressions
+        expect(await parentTaskField.isExisting()).to.be.false
+      }
+    })
+
+    it("We should see list of children tasks if there are any related to the current task", async() => {
+      const childrenTasks = await (await ShowTask.getChildrenTasks())
+      const childrenTasksField = await (await ShowTask.getChildrenTasksField())
+      if (await childrenTasks.isExisting()) {
+        await (await ShowTask.getChildrenTasksField()).waitForDisplayed()
+        // eslint-disable-next-line no-unused-expressions
+        expect(await childrenTasksField.isExisting()).to.be.true
+      }
+    })
+
+    it("We should not see list of children tasks if there are not any related to the current task", async() => {
```

Comment:
Change this test to "When in the show page of a task without a children tasks", load a task without children tasks, and do the appropriate assertion.